accounting/accounting.go

```diff
@@ -1,119 +1,45 @@
 package accounting
 
 import (
-	"bufio"
-	"encoding/csv"
 	"log"
-	"os"
-	"strings"
+	"net/http"
 
-	lib "github.com/wopta/goworkspace/lib"
+	"github.com/GoogleCloudPlatform/functions-framework-go/functions"
+	"github.com/wopta/goworkspace/lib"
+	"github.com/wopta/goworkspace/models"
 )
 
-func main() {
-	GetFx()
+func init() {
+	log.Println("INIT Accounting")
+	functions.HTTP("Accounting", Accounting)
 }
-func csvL() {
 
-	//dfres := dataframe.NewDataFrame()
-	//dfres := lib.NewDf()
-	if _, err := os.Stat("output.csv"); err == nil {
-		log.Printf("File exists\n")
-		e := os.Remove("output.csv")
-		lib.CheckError(e)
-	} else {
-		log.Printf("File does not exist\n")
+func Accounting(w http.ResponseWriter, r *http.Request) {
+	log.Println("Accounting")
+	lib.EnableCors(&w, r)
+
+	route := lib.RouteData{
+		Routes: []lib.Route{
+			{
+				Route:   "/network/transactions/v1/transaction/:transactionUid",
+				Handler: GetNetworkTransactionsFx,
+				Method:  http.MethodGet,
+				Roles:   []string{models.UserRoleAdmin, models.UserRoleManager},
+			},
+			{
+				Route:   "/network/transactions/v1/:uid",
```

Comment:
should be changed to `/network/transactions/v1/transaction/:transactionUid` in order to set a standard with other endpoints in domain